### PR TITLE
Atomizes smithing a bit and fixed some recipes not showing.

### LINKED ISSUE #1465: [BUG] Blacksteel cant be smithed ?
## Describe the bug
cant hit a blacksteel bar on the anvil

## Round ID:
image

## Steps to Reproduce:
image
put blacksteel on an anvil, try to hit it with a hammer

## Additional context
likely related to https://github.com/Rotwood-Vale/Ratwood-Keep/pull/1353/files

### PR DESCRIPTION
**Pull request by TheFenix323**
## About The Pull Request
Now the categories are given by the parent type, doing this also fixed recipes like Blacksteel armors not showing up.

## Iron now has a engineering tab
iron cat
iron engi

## Bronze now shows the weapons tab
Bonze cat
bronze weap

## Blacksteel now shows an armor tab
blacksteel cat
blacksteel armor

## Why It's Good For The Game
Makes the code more tidy and fixes #1465 now has a engineering tab

### CODE REVIEW

**Comment by Nakalon** [comment]
My love, this is absolutely astonishing, but may you wait on my Artificer PR? I deleted the engineering file and don't want to have to conflit resolve a forth time 😭

**Comment by TheFenix323** [comment]
> My love, this is absolutely astonishing, but may you wait on my Artificer PR? I deleted the engineering file and don't want to have to conflit resolve a forth time 😭 

For sure, I'll update it when your PR is merged

**Comment by TheFenix323** [comment]
> My love, this is absolutely astonishing, but may you wait on my Artificer PR? I deleted the engineering file and don't want to have to conflit resolve a forth time 😭

Merge conflicts solved

**Comment by Nakalon** [comment]
> > My love, this is absolutely astonishing, but may you wait on my Artificer PR? I deleted the engineering file and don't want to have to conflit resolve a forth time 😭
> 
> Merge conflicts solved

love you

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm [hunk 1]
```diff
@@ -127,54 +114,39 @@
...
-	i_type = "Valuables"
 
 /datum/anvil_recipe/valuables/silver/diamond
 	name = "Dorpel Ring (+1 Dorpel)"
 	req_bar = /obj/item/ingot/silver
 	additional_items = list(/obj/item/roguegem/diamond)	
 	created_item = /obj/item/clothing/ring/diamonds
-	i_type = "Valuables"
-
-/datum/anvil_recipe/valuables/terminus
```
Comment (Nakalon) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm:167]: this was so silly by gob

**Review by Nakalon — APPROVED** [review]
bery nice

**Comment by Nakalon** [comment]
Except the engineering tab is gone 👁️

**Pull request merged**